package.json

```diff
@@ -1,82 +1,108 @@
+        "t3": "cd tests/serialize && node ./data.js -in=t3.ttl -format=application/rdf+xml  -out=,t3.xml && jsdiff ,t3.xml t3-ref.xml",
+        "t4": "cd tests/serialize && node ./data.js -in=t3.ttl -out=,t4.ttl && jsdiff ,t4.ttl t4-ref.ttl",
+        "t5": "cd tests/serialize && node ./data.js -in=t5.n3 -format=text/turtle -out=,t5.ttl && jsdiff ,t5.ttl t5-ref.ttl",
+        "t6": "cd tests/serialize && node ./data.js -in=t5.n3 -format=text/n3 -out=,t6.n3 && jsdiff ,t6.n3 t6-ref.n3",
+        "t7": "cd tests/serialize && node ./data.js -in=t7.n3 -format=application/n-triples -out=,t7.nt && jsdiff ,t7.nt t7-ref.nt",
+        "t8": "cd tests/serialize && node ./data.js -in=t5.n3  -format=application/n-quads -dump=,t8.nq && jsdiff ,t8.nq t8-ref.nq",
+        "t9": "cd tests/serialize && node ./data.js -in=t7.n3 -format=application/ld+json -out=,t9.json && jsdiff ,t9.json t9-ref.json"
+    },
+    "files": [
+        "dist",
+        "dist/rdflib.js",
+        "lib",
+        "src"
+    ],
+    "main": "lib/index.js",
+    "keywords": [
+        "linkeddata",
+        "linked data",
+        "rdf",
+        "rdfa",
+        "turtle",
+        "semantic",
+        "web",
+        "read-write web"
+    ],
+    "standard": {
+        "globals": [
+            "ActiveXObject",
+            "Components",
+            "DOMParser",
+            "netscape",
+            "$rdf",
+            "Services",
+            "tabulator",
+            "WebSocket"
+        ]
     }
-  ],
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/linkeddata/rdflib.js.git"
-  },
-  "homepage": "http://github.com/linkeddata/rdflib.js",
-  "bugs": "http://github.com/linkeddata/rdflib.js/issues",
-  "dependencies": {
-    "async": "^0.9.x",
-    "jsonld": "^0.4.5",
-    "n3": "^0.4.1",
-    "xmldom": "^0.1.22",
-    "xmlhttprequest": "^1.7.0"
-  },
-  "devDependencies": {
-    "babel-cli": "^6.11.4",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-register": "^6.18.0",
-    "babelify": "^7.3.0",
-    "browserify": "^13.0.0",
-    "chai": "^3.5.0",
-    "minifyify": "^7.3.2",
-    "mocha": "^3.2.0"
-  },
-  "scripts": {
-    "test": "npm run test:unit && make test",
-    "test:unit": "mocha --growl --require babel-register tests/unit/**-test.js",
-    "test:unit:dev": "mocha --watch --growl --require babel-register tests/unit/**-test.js",
-    "prepublish": "make lib"
-  },
-  "files": [
-    "dist",
-    "dist/rdflib.js",
-    "lib",
-    "src"
-  ],
-  "main": "lib/index.js",
-  "keywords": [
-    "linkeddata",
-    "linked data",
-    "rdf",
-    "rdfa",
-    "turtle",
-    "semantic",
-    "web",
-    "read-write web"
-  ],
-  "standard": {
-    "globals": [
-      "ActiveXObject",
-      "Components",
-      "DOMParser",
-      "netscape",
-      "$rdf",
-      "Services",
-      "tabulator",
-      "WebSocket"
-    ]
-  }
-}
+}
```

Comment:
Might be my nitpicky nature, but I'd prefer if we kept our final newlines in all files.